UI/MapWnd.cpp

```diff
@@ -1816,7 +1824,7 @@ void MapWnd::Render() {
 }
 
 void MapWnd::RenderStarfields() {
-    if (!GetOptionsDB().Get<bool>("UI.galaxy-starfields"))
+    if (!GetOptionsDB().Get<bool>("ui.map.system.background.starfields.shown"))
```

Comment:
this option names doesn't make sense... background starfields have nothing to do with systems